server/mlabns/tests/test_production_check.py

```diff
@@ -0,0 +1,45 @@
+import unittest
+
+from mlabns.util import production_check as pc
+
+
+class CheckProductionTestCase(unittest.TestCase):
+
+    def testIsProductionSite(self):
+        self.assertTrue(pc.is_production_site('nuq01'))
```

Comment:
`zzz09` should pass because it fits the site schema. All mlab-ns knows is what it sees in ks and what it knows about the schema. If ks says there's a site called `zzz09` then mlab-ns should treat that site as a production site because a) ks reported it and b) it fits the production site schema.

Given this, do you feel like it provides test coverage that the other positive test cases don't provide?